crates/typst/src/model/content.rs

```diff
@@ -704,3 +725,52 @@ fn missing_field_no_default(field: &str) -> EcoString {
         field.repr()
     )
 }
+
+/// Makes sure the content is not shared and returns a mutable reference to the
+/// inner element.
+#[doc(hidden)]
+#[allow(invalid_value)]
```

Comment:
```suggestion
```

Reply:
Right that's no longer needed, nice catch!